### PR TITLE
[16.0][MIG]sales_report_product_image:Migrated module.

### PR DESCRIPTION
**Pull request by maitree-serpentcs**
**Migrated by: Bhavesh Patil**

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
@maitree-serpentcs did we review? Compared this with v16?
- Are not we adding new features ?

**Comment by maitree-serpentcs** [comment]
I have reviewed code and added PR.
Module is for adding image for products in sales report so there are not any other features we are going to add.
I have send module for testing.

**Comment by JayVora-SerpentCS** [comment]
@pragatipatel-serpentcs plz review.

File: sales_report_product_image/tests/test_sales_report_product_image.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2016-TODAY Serpent Consulting Services Pvt. Ltd.
+# See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+
+
+class SaleReportProductImageTestCase(common.TransactionCase):
+
+    def setup(self):
```
Comment (pragatipatel-serpentcs) [inline sales_report_product_image/tests/test_sales_report_product_image.py:9]: Can we remove this? as no extra code is written.

**Comment by maitree-serpentcs** [comment]
Testing done by Sidhharaj

**Pull request merged**